packages/trs/trs.1.0.0/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+synopsis:
+  "Parsing library for the TRS format, the format for first-order rewrite systems"
+description:
+  "trs is a parsing library for the TRS format, the format for first-order rewrite systems."
+maintainer: ["jajima.jp@gmail.com"]
+authors: ["Soichi Yajima"]
+license: "MIT"
+tags: ["trs" "rewrite systems"]
+homepage: "https://github.com/jajimajp/trs"
+bug-reports: "https://github.com/jajimajp/trs/issues"
+depends: [
+  "ocaml" {>= "4.12.0"}
+  "dune" {>= "3.7" & >= "3.7"}
```

Comment:
```suggestion
  "dune" {>= "3.7"}
```
To avoid this in the future, please remove dune from the dependencies in the dune-project file (it will add itself automatically with the correct bound when generating the opam file)